app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-print-intent-tracker.js

```diff
@@ -0,0 +1,24 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analyticsModules || {};
+
+(function (analyticsModules) {
+  'use strict'
+
+  var Ga4PrintIntentTracker = {
+    init: function () {
+      window.addEventListener('beforeprint', function () {
+        var data = {
+          event_name: 'print_page',
+          type: 'print page',
+          method: 'browser print'
+        }
+        var schemas = new window.GOVUK.analyticsGa4.Schemas()
+        var schema = schemas.mergeProperties(data, 'event_data')
```

Comment:
Thought for a moment that you'd declared the same variable twice! But you hadn't. Is it worth reusing the schemas variable here?

Side note - possible bit of tech debt. I think always now we do these three lines (declare the schema, merge the schema, call sendData) and maybe we could move all of that into sendData and just call that instead. I'll look into it.

Reply:
Have added a card: https://trello.com/c/sI76EaNX/655-investigate-shortening-calls-to-schema-code

Reply:
Sounds good :+1: